projects/packages/forms/src/icons/akismet-icon.js

```diff
@@ -0,0 +1,35 @@
+import { SVG } from '@wordpress/primitives';
+
+const AkismetIcon = props => (
```

Comment:
We should have Akismet logo under components, see `projects/js-packages/components/components/icons` and look for "Anti Spam".